### ISSUE TITLE
Not able to preview the app 

### ISSUE THREAD

**Issue by Voracious08** [fromtexas/react-native-budget-app#2]
The version of the SDK is 31.00. To preview it should be above 42.00.

**Comment by fromtexas** [comment]
Yeah, it hasn't been updated for more than 3 years and it has a lot of outdated dependencies ¯\\\_(ツ)_/¯
